zingolib/src/error.rs

```diff
@@ -35,6 +35,8 @@ pub enum ZingoLibError {
     /// TODO: Add Doc Comment Here!
     MissingOutputIndex(TxId),
     /// TODO: Add Doc Comment Here!
+    MissingOutputIndexInThisTx,
```

Comment:
Where is the `MissingOutputIndex` variant used?

Reply:
decided not to use it